### PR TITLE
[IMP] l10n_eu_service: OSS - Automatic Tax and Fiscal Position Mapping for EU Countries

### PR DESCRIPTION
**Pull request by oco-odoo**
13.0 backport of https://github.com/odoo/odoo/commit/d347eae1646639e2b346d404d1eb68ce9eac005c

Due to EU e-commerce 'One-Stop Shop' which becomes available on 1 July 2021, EU companies involved in distance sales are able to file their taxes using the new OSS process. Adopting OSS is now simplified in Odoo.

On installation of the module, all companies having a fiscal country in the EU will be processed.
For all existing domestic taxes in the company, an associated foreign tax (standard rate, reduced rate, etc) is found in the `EU_TAX_MAP`. This mapping will be created in a fiscal position that is automatically detected based on the customers' country.

All that is required from the user, is to review the tax mappings according to the products and services sold by the company.

A refresh button is also available in the odoo settings to redo/update the fiscal positions. This might be useful after adding a new tax.

Note: The tax mapping herein is not intended to cover all possible tax mappings between EU countries, but rather, the most commonly used ones. It is advised that users with special tax cases update the created Fiscal Positions according to their requirements.

Closes [Task 2579615]

### CODE REVIEW

**Comment by oco-odoo** [comment]
@fw-bot disable
(it's already in 14.0)

**Comment by oco-odoo** [comment]
This also contains the fix done here https://github.com/odoo/odoo/pull/73078/files

File: addons/l10n_eu_service/wizard/wizard.xml [hunk 1]
```diff
@@ -49,7 +49,4 @@
             <field name="target">new</field>
         </record>
 
-        <record id="open_l10n_eu_service" model="ir.actions.todo">
-            <field name="action_id" ref="action_eu_service"/>
-        </record>
```
Comment (smetl) [inline addons/l10n_eu_service/wizard/wizard.xml:54]: I'm not sure you can remove this in stable without risking anything.

**Comment by smetl** [comment]
@robodoo r+

**Pull request closed without merging**